types/ramda/index.d.ts

```diff
@@ -76,12 +75,6 @@ import {
 
 export * from './tools';
 
-/**
- * Placeholder. When used with functions like curry, or op, the second argument is applied to the second
- * position, and it returns a function waiting for its first argument.
- */
-export const __: Placeholder;
```

Comment:
I would recommend putting the removal of `Placeholder` into a separate PR since its removal is more controversial than contains/merge.

Reply:
OK, I'll split this pr into 2 separate PRs.